src/nodes/display/RenderOutputNode.js

```diff
@@ -58,7 +58,7 @@ class RenderOutputNode extends TempNode {
 		/**
 		 * The tone mapping type.
 		 *
-		 * @type {Number?}
+		 * @type {String?}
```

Comment:
The ctor needs an update since the value behind constants like `THREE.LinearToneMapping` are numbers.

Reply:
Then it might be the constructor parameter type that is wrong ? 

```@param {String} toneMapping - The tone mapping type.```

Reply:
Yes, the ctor needs an update.